react-app/src/components/CodeListing.tsx

```diff
@@ -0,0 +1,27 @@
+import { CSSProperties } from "react";
+import styles from "../styles/GridPage.module.css";
+
+export function CodeListing({ cssProps }: { cssProps: CSSProperties }) {
```

Comment:
we need this component to be more flexible, the flexbox page also uses it

Reply:
I will make it in a new PR